common/lanelet2_extension/lib/CarmaTrafficSignal.cpp

```diff
@@ -60,6 +60,42 @@ LineStrings3d CarmaTrafficSignal::stopLine()
   return getParameters<LineString3d>(RoleName::RefLine);
 }
 
+Optional<ConstLineString3d> CarmaTrafficSignal::getStopLine(const ConstLanelet& llt) const 
+{
+  auto sl = stopLine();
+  if (sl.empty()) {
+    return {};
```

Comment:
This isn't the same as returning an empty optional I don't think. 
Is ```boost::none``` what is intended here? (same below)

Reply:
used boost::none